### PR TITLE
Locked airlocks

### PR DESCRIPTION
**Pull request by AJCM-git**
- Adds locked airlocks with their respective access
- Configures jobs access correctly
- Organizes and cleans up airlock-related yamls
- Deletes airlocks whitout access from saltern 

NOTE: Might conflict with #982

### CODE REVIEW

File: Content.Client/GameObjects/Components/Access/IdCardConsoleWindow.cs [hunk 1]
```diff
@@ -132,15 +132,15 @@ public IdCardConsoleWindow(IdCardConsoleBoundUserInterface owner, ILocalizationM
                 };
                 vBox.AddChild(grid);
 
-                foreach (var accessLevel in prototypeManager.EnumeratePrototypes<AccessLevelPrototype>())
+                foreach (var AccessLevel in prototypeManager.EnumeratePrototypes<AccessLevelPrototype>())
```
Comment (PJB3005) [inline Content.Client/GameObjects/Components/Access/IdCardConsoleWindow.cs:135]: This rename is bad, the file was already correctly capitalized.

File: Content.Shared/Access/AccessLevelPrototype.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ namespace Content.Shared.Access
     /// <summary>
     ///     Defines a single access level that can be stored on ID cards and checked for.
     /// </summary>
-    [Prototype("accessLevel")]
+    [Prototype("AccessLevel")]
```
Comment (PJB3005) [inline Content.Shared/Access/AccessLevelPrototype.cs:11]: Prototypes are camelCase, don't rename this.

File: Resources/Prototypes/Entities/Buildings/Doors/airlock_access.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Current access: "Maintenance", "Cargo", "Engineering", "Medical", "Research", "Security", "Service", "Captain", "Command", "External"
+- type: entity
+  parent: Airlock
+  id: AirlockLocked
+  suffix: Locked
+  components:
+  - type: AccessReader
+    necessary: ["Maintenance"]
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Buildings/Doors/airlock_access.yml:8]: Why is maintenance the access level for this "plain" airlock?

File: Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml [hunk 4]
```diff
@@ -49,16 +49,13 @@
 - type: entity
   id: AirlockGlass
   parent: Airlock
-  name: Glass Airlock
+  name: Glass airlock
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml:52]: I mean if you're gonna rename this name to be lower case at least do it fully..

File: Resources/Prototypes/Entities/Markers/spawn_points_jobs.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Base
 - type: entity
   id: SpawnPointJobBase
-  parent: spawn_point
+  parent: SpawnPoint
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Markers/spawn_points_jobs.yml:4]: You need to update stationstation as well if you're gonna change this.

File: Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml (hunk 4 above)
Comment (AJCM-git) [reply Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml:52]: didnt we agreed on using upper case for the first word and lower case for the rest when naming?

File: Resources/Prototypes/Entities/Markers/spawn_points_jobs.yml (hunk 5 above)
Comment (AJCM-git) [reply Resources/Prototypes/Entities/Markers/spawn_points_jobs.yml:4]: Done

File: Resources/Prototypes/Entities/Buildings/Doors/airlock_access.yml (hunk 3 above)
Comment (AJCM-git) [reply Resources/Prototypes/Entities/Buildings/Doors/airlock_access.yml]: I thought we were going to use it as a placeholder for maint doors

File: Content.Shared/Access/AccessLevelPrototype.cs (hunk 2 above)
Comment (AJCM-git) [reply Content.Shared/Access/AccessLevelPrototype.cs]: Done

File: Content.Client/GameObjects/Components/Access/IdCardConsoleWindow.cs (hunk 1 above)
Comment (AJCM-git) [reply Content.Client/GameObjects/Components/Access/IdCardConsoleWindow.cs]: Done

File: Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml (hunk 4 above)
Comment (PJB3005) [reply Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml:58]: Uh, no?

File: Content.Shared/Access/AccessLevelPrototype.cs (hunk 2 above)
Comment (PJB3005) [reply Content.Shared/Access/AccessLevelPrototype.cs]: You need to actually update all the prototypes you changed now to use the correct naming again.

**Pull request merged**